cmd/pebble/cmd_warnings.go

```diff
@@ -56,7 +56,7 @@ var shortOkayHelp = "Acknowledge warnings"
 var longOkayHelp = `
 The okay command acknowledges the warnings listed with 'pebble warnings'.
 
-Once acknowledged a warning won't appear again unless it re-occurrs and
+Once acknowledged a warning won't appear again unless it reoccurs and
```

Comment:
Drive-by fix: "reoccur" is a word in its own right without a hyphen ([ref](https://www.merriam-webster.com/dictionary/reoccur)).

Reply:
While you're here, I'd put a comma after "acknowledged".